### ISSUE TITLE
1.20.1 Update crashes when used with Supplementaries mod 

### ISSUE THREAD

**Issue by Orangeinferno** [ObliviousSpartan/SpartanShields#3]
tried loading into single player with the mod on my modpack, it crashes instantly. The crash says the following.

The game crashed whilst exception in server tick loop
Error: net.mehvahdjukaar.supplementaries.common.utils.VibeChecker$BadModError: Some OTHER mod loaded the Sheets class to early, causing modded banner patterns and sherds textures to not include modded ones.

it is specifically this mod. I have no idea why it does this.

**Comment by SuperStefan97** [comment]
Hi! I have this same bug, mistakenly I poste it on the Spartan Weaponry repository. Here's the link to that issue with the crash and .latest logs.
https://github.com/ObliviousSpartan/SpartanWeaponry/issues/43#issue-2353776443
Also, it seems it has something to do with the way banners are rendered, Supps changes some of this things to make flags and hanging banners. On the crash log it explains (or do it seems to be mere mortal) how to fix it.
Also, just using Spartan Weaponry does not cause trouble.
Thanks for porting this awesome mod!

**Comment by PhilTheGreatLOL** [comment]
[Uploading crash-2024-06-15_15.28.25-server.txt…]()
Same, Impact for me too.

**Comment by yMemez** [comment]
I got the same issue.

I found this in the logs:
```
[15:19:34] [modloading-worker-0/ERROR]: net.minecraft.client.renderer.Sheets loaded too early, modded registry-based materials may not work correctly
java.lang.IllegalStateException: net.minecraft.client.renderer.Sheets loaded too early
	at net.minecraft.client.renderer.Sheets.<clinit>(Sheets.java:226) ~[client-1.20.1-20230612.114412-srg.jar%23764!/:?]
	at com.oblivioussp.spartanshields.client.render.item.TowerShieldRenderInfo.<init>(TowerShieldRenderInfo.java:22) ~[SpartanShields-1.20.1-forge-3.1.0.jar%23718!/:1.20.1-forge-3.1.0]
	at com.oblivioussp.spartanshields.client.render.item.TextureStitcher.<clinit>(TextureStitcher.java:12) ~[SpartanShields-1.20.1-forge-3.1.0.jar%23718!/:1.20.1-forge-3.1.0]
	at java.lang.Class.forName0(Native Method) ~[?:?]
	at java.lang.Class.forName(Class.java:467) ~[?:?]
	at net.minecraftforge.fml.javafmlmod.AutomaticEventSubscriber.lambda$inject$6(AutomaticEventSubscriber.java:61) ~[javafmllanguage-1.20.1-47.3.1.jar%23766!/:?]
	at java.util.ArrayList.forEach(ArrayList.java:1511) ~[?:?]
	at net.minecraftforge.fml.javafmlmod.AutomaticEventSubscriber.inject(AutomaticEventSubscriber.java:48) ~[javafmllanguage-1.20.1-47.3.1.jar%23766!/:?]
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:85) ~[javafmllanguage-1.20.1-47.3.1.jar%23766!/:?]
	at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$4(ModContainer.java:124) ~[fmlcore-1.20.1-47.3.1.jar%23765!/:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796) ~[?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?]
```
And this after joining the world and it crashing:
```
[15:22:49] [Server thread/ERROR]: Encountered an unexpected exception
net.mehvahdjukaar.supplementaries.common.utils.VibeChecker$BadModError: Some OTHER mod loaded the Sheets class to early, causing modded banner patterns and sherds textures to not include modded ones.
```

**Comment by ObliviousSpartan** [comment]
Thanks for the reports,
I'd have had this fixed earlier but the day after uploading the new versions of both my mods, I fell ill and wasn't able to do anything for a couple of days.
I've looked at what was causing this issue and found this was the offending line:
https://github.com/ObliviousSpartan/SpartanShields/blob/8442742232746c0fe8a2d33548628f1d251c338a/src/main/java/com/oblivioussp/spartanshields/client/render/item/TextureStitcher.java#L9
This line should've been removed before I released it! Sorry about that.
I will upload that fix shortly.
